pytket/extensions/qiskit/backends/aer.py

```diff
@@ -236,7 +226,7 @@ def _arch_independent_default_compilation_pass(
         return SequencePass([DecomposeBoxes(), FullPeepholeOptimise()])
 
     def default_compilation_pass(
-        self, optimisation_level: int = 2, placement_options: Optional[Dict] = None
+        self, optimisation_level: int = 2, placement_options: Optional[dict] = None
```

Comment:
These `Optional[dict]` annotations are actually incomplete. We should update the types of the keys and values in a separate PR.

Reply:
fixed the incomplete dict typing in [8ee1d85](https://github.com/CQCL/pytket-qiskit/pull/383/commits/8ee1d852d41e743c532d9df0cf44fbef92b1f900) and [d93d126](https://github.com/CQCL/pytket-qiskit/pull/383/commits/d93d126a9876baae22b1c3e448ebf37340722737)